wallet-connect-test-bench/README.md

```diff
@@ -0,0 +1,41 @@
+# Test Bench
+
+A test bench for testing mobile wallets (via walletConnect) or the browser wallet.
+
+## Prerequisites
+
+-   Browser wallet extension must be installed in Google Chrome browser and the Concordium testnet needs to be selected or a mobile wallet needs to be set up that supports walletConnect in order to view smart contract details or submit transactions.
+
+## Running the test bench front-end
+
+Clone the repo:
+
+```shell
+git clone --recursive-submodules git@github.com:Concordium/concordium-misc-tools
+```
+
+Navigate into ./deps/concordium-dapplibraries and build the dapp libraries packages:
+
+```shell
+cd ./deps/concordium-dapp-libraries/
+yarn
+yarn build
+```
+
+Navigate into this folder:
+```shell
+cd ./wallet-connect-test-bench
+```
+
+-   Run `yarn install` in this folder.
+-   Run `cp -r ../deps/concordium-dapp-libraries/packages/react-components ./node_modules/@concordium/react-components` in this folder.
+-   Run `cp -r ../deps/concordium-dapp-libraries/packages/wallet-connectors ./node_modules/@concordium/wallet-connectors` in this folder.
+-   Run `yarn build` in a terminal in this folder.
+-   Run `yarn start`.
+-   Open URL logged in console (typically http://127.0.0.1:8080).
```

Comment:
I added the following command into the `package.json` file to make this less manual.

> "preinstall": "cp -r ../../deps/concordium-dapp-libraries/packages/react-components ./node_modules/@concordium/react-components; cp -r ../../deps/concordium-dapp-libraries/packages/wallet-connectors ./node_modules/@concordium/wallet-connectors",

Alternatively:
I experiment a bit with local dependencies but that caused additional `node_modules` folders to appear in the `react-components/wallet-connectors` package directory causing problems (meaning the installed local packages path looked something like 'node_modules/@concordium/react-components/node_modules/@concordium...').
```
        "@concordium/react-components": "../../deps/concordium-dapp-libraries/packages/react-components",
        "@concordium/wallet-connectors": "../../deps/concordium-dapp-libraries/packages/wallet-connectors",
```

 Any idea about this local dependencies issue, otherwise above `preinstall` steps seems to work fine for me.

Reply:
I had to create the directory manually, otherwise it does indeed work fine. Perhaps adding 
```
mkdir -p ./node_modules/@concordium/;
``` to the preinstall step would be helpful.

Reply:
hmm, the folder `./node_modules/@concordium/` should have already been created if you follow the steps in the `ReadMe.mk`. The `yarn/yarn install` command installs some dependencies from `@concordium` so there should be this folder already.

Reply:
Ok.